index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
+    <title>Document</title>
+</head>
+<body>
+    <header>
+        <div class="head-content">
+            <p class="logo">
+               <a href="http//">Welcome</a>
+            </p>
+            <p class="nav">
+                <span></span>
+                <span></span>
+                <span></span>
+            </p>
+        </div>
+    </header>
+    <section class="presentation">
+    <div class="content">
+            <h1> Hey There, I'm Ghislain </h1>
+            <h1 class="intros">I'm a software developer</h1>
+        <p class="text">I can help you build a product, feature or website Look through some of my Work and experience ! If you like what you see and you need coded,don't hestiate to contact me.</p>
+        <p class="icons">
+            <a href=""><i class="fa-brands fa-twitter"></i></a>
+            <a href=""><i class="fa-brands fa-linkedin-in"></i></a>
+            <a href=""><i class="fa-brands fa-github"></i></a>
+            <a href=""><i class="fa-solid fa-m"></i></a>
+            <a href=""><i class="fa-brands fa-angellist"></i></a>
+        </p>
```

Comment:
- [ ] Since this is a list, the semantically appropriate tag to use here is `<ul>`. And why not `<ol>`? because we don't care about their order. right? Therefore, kindly use use something like this:
```
<ul>
   <li>
     <a href=""><i class="fa-brands fa-linkedin-in"></i></a>
   </li>
   <li>
     <a href=""><i class="fa-brands fa-github"></i></a>
   </li>
   ....
</ul>
```